docker/dev/Dockerfile

```diff
@@ -10,27 +10,36 @@ RUN yarn install --pure-lockfile
 COPY ui /app/ui
 RUN yarn build
 
-FROM ekidd/rust-musl-builder:1.42.0-openssl11 as rust
+FROM ekidd/rust-musl-builder:nightly-2020-05-07 as rust
```

Comment:
It needs nightly, because it uses some unstable features to build the dependency tree.

Reply:
This isnt ideal, our CI should ensure that Lemmy builds fine on latest stable and doesnt use any nightly only features. Maybe we need to add a separate Dockerfile for CI?